src/main/java/core/basesyntax/strategy/SupplyHandler.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public class SupplyHandler implements OperationHandler {
+    private static final String SUPPLY = "s";
```

Comment:
The constant 'SUPPLY' should be private if it's not used outside of this class. Verify that its access modifier is set correctly.